announcer.go

```diff
@@ -39,21 +38,23 @@ type playlist struct {
 func main() {
 	// Entry point to the application
 	// Boilerplate CLI stuff
-	app := cli.NewApp()
-	app.Name = "Announcer"
-	app.Usage = "Report current track to the Prostream"
-	app.Flags = []cli.Flag{
-		cli.StringFlag{Name:"prostream",  Value:"", Usage:"IP address or hostname of the Prostream device."},
-		cli.IntFlag{Name:"port", Value:9000, Usage:"Port of the Prostream track information receiver"},
-		cli.StringFlag{Name:"chirp", Value:"https://chirpradio.appspot.com/api/current_playlist", Usage:"URL of the CHIRP current_playlist API endpoint"},
-		cli.BoolFlag{Name:"verbose", Usage:"Run in Verbose mode."},
-		cli.BoolFlag{Name:"test", Usage:"Run in test mode. Sends nothing to Prostream"},
-		cli.BoolFlag{Name:"runOnce", Usage:"Run once and then quit"},
-		cli.StringFlag{Name:"rds", Value:"", Usage:"IP address or hostname of the RDS Encoder"},
-		cli.IntFlag{Name:"pdsPort", Value:23, Usage:"Port used by the RDS Encoder"},
-	}
-	app.Action = func(c *cli.Context) {
-		parseAndRun(c)
+	app := &cli.App{
```

Comment:
Change here represent an update in the `github.com/urfave/cli` library, and the "dps" "rds" typo.